finnflow/src/main/java/no/finn/android/finnflow/ParcelableParcer.java

```diff
@@ -0,0 +1,15 @@
+package no.finn.android.finnflow;
+
+import android.os.Parcelable;
+
+public class ParcelableParcer implements flow.StateParceler {
+    @Override
+    public Parcelable wrap(Object instance) {
+        return (Parcelable) instance;
+    }
+
+    @Override
+    public Object unwrap(Parcelable parcelable) {
+        return parcelable;
```

Comment:
Should atleast leave a comment here about how to use a wrapper to support for example jackson/gson etc? Can contain just example code for serialization, but it should be mentioned how this is easy to do.